gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -116,6 +125,35 @@ def main(args):
                     )
             logger.info("Done with preprocessing genome and exome Table.")
 
+        mutation_ht = mutation_rate_ht.versions[version].ht().select("mu_snp")
+
+        # Create training dataset that includes possible and observed variant counts for building models.
+        if args.create_training_set:
+            logger.info("Counting possible and observed variant counts...")
+
```

Comment:
See other PR for making `preprocess_resources` and `training_resources`
```suggestion
            # Check if the input/output resources exist.
            check_resource_existence(
                "--preprocess-data",
                "--create-training-set",
                preprocess_resources.values(),
                training_resources.values(),
                overwrite,
            )
```